Ghidra/Features/WildcardAssembler/ghidra_scripts/WildSleighAssemblerInfo.java

```diff
@@ -0,0 +1,157 @@
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// Prints information about the results of assembling an instruction using the WildSleighAssembler
+// when that instruction has one or more wildcards in it.
+//
+// This script uses currentProgram and currentAddress to determine architecture and location.
+//
+// Notice that this script doesn't only output the assembled bytes of an instruction, but also more
+// specific information about each wildcard in the input instruction.
+//
+// See the "FindInstructionWithWildcard" script for another example of using the WildSleighAssembler
+// @category Examples
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import ghidra.app.plugin.assembler.AssemblySelector;
+import ghidra.app.plugin.assembler.sleigh.parse.AssemblyParseResult;
+import ghidra.app.plugin.assembler.sleigh.sem.AssemblyResolution;
+import ghidra.app.plugin.processors.sleigh.SleighLanguage;
+import ghidra.app.script.GhidraScript;
+import ghidra.asm.wild.WildOperandInfo;
+import ghidra.asm.wild.WildSleighAssembler;
+import ghidra.asm.wild.WildSleighAssemblerBuilder;
+import ghidra.asm.wild.sem.WildAssemblyResolvedPatterns;
+
+public class WildSleighAssemblerInfo extends GhidraScript {
+
+	List<String> sampleInstructions =
+		Arrays.asList("MOV EAX,`Q1`", "MOV RDI,qword ptr [`Q1` + -0x30]", "Custom");
+
+	public void run() throws Exception {
+
+		String instruction = askChoice("Instruction to assemble", "Assemble this instruction:",
+			sampleInstructions, "Custom");
+
+		if (instruction.equals("Custom")) {
+			instruction = askString("Instruction",
+				"Instruction to assemble and print information about.",
+				"MOV RDI,qword ptr [`Q1` + -0x30]");
+		}
+		var assemblyResolutions = getAllAssemblyResolutions(instruction);
+		printAssemblyParseResults(assemblyResolutions);
+	}
+
+	/**
+	 * Use a WildSleighAssembler to assemble the given {@code wildcardedInstruction}
+	 * 
+	 * @param wildcardedInstruction
+	 * @return All AssemblyParseResult produced from the given input
+	 */
+	private List<AssemblyResolution> getAllAssemblyResolutions(
+			String wildcardedInstruction) {
+
+		SleighLanguage language = (SleighLanguage) currentProgram.getLanguage();
+
+		// Make sure that if one of the example instructions was chosen the current binary has the
+		// correct architecture.
+		if (sampleInstructions.contains(wildcardedInstruction) &&
+			!language.getLanguageID().toString().equals("x86:LE:64:default")) {
```

Comment:
Ditto from the other script.

Reply:
This is different from the other script because here I am just raising a warning if the user is using one of the example instructions with an unexpected architecture). I've removed the "return" statement right after this to allow us to try anyways (and updated the warning text slightly). Do you think I should do something different here?

Reply:
Oh. Honestly, I missed the `if` statement on Line 75. I'll re-read as I check your other revisions.